amundsen_application/static/js/components/HomePage/styles.scss

```diff
@@ -0,0 +1,13 @@
+@import 'variables';
+
+.home-page {
```

Comment:
Would be curious to see a screenshot of what this looks like -- is the first `SearchBar` pushed further down on this page than it is on the `SearchPage`? Is the `BookmarkList` further away fro the `SearchBar` on this page as well?

Reply:
oh whoops shouldn't be applying the home-element-container class to SearchBar, will attach a screenshot of the fixed version

Reply:
<img width="1294" alt="Screen Shot 2019-06-11 at 2 21 45 PM" src="https://user-images.githubusercontent.com/8528315/59307866-48eace00-8c54-11e9-8e2d-a0413c375114.png">